### PR TITLE
[Docs] Update Azure readme

### PR DESCRIPTION
**Pull request by J-467**
- Update Azure readme on local testing
- update the prerequisite to include the install make command

### CODE REVIEW

File: pkg/azure/README.md [hunk 1]
```diff
@@ -84,24 +95,24 @@ Follow these steps to ensure consistent structure and seamless integration.
...
+     handler, err = s.setupAzureHandler(resourceIdInfo, req.Deployment.Namespace)
      if err != nil {
          return nil, err
      }
+     // Use handler
      ```
 
 3. Handling ARM Requests:
-   - If your new API needs to make ARM requests, check if the required functionality is already implemented in `sdk_handler.go`. If not, add a new function in the handler that wraps the Azure request. Call this new function from the `plugin` file. Any helper functionalities relevant only to Paraglider and the server can remain in `plugin`.
+   - If your new API needs to make ARM requests, check if the required functionality is already implemented in `sdk_handler.go`. If not, add a new function in the handler that wraps the Azure request. Call this new function from the `plugin` file. Ideally, helper functionalities relevant to Paraglider and the server should be outside `plugin`.
```
Comment (smcclure20) [inline pkg/azure/README.md:107]: Could we say "Ideally, only the Paraglider plugin interface RPCs should be in `plugin` and helper functions should go in the appropriate file" or something like that?

**Review by smcclure20 — CHANGES_REQUESTED** [review]
Thanks for updating this!

File: pkg/azure/README.md (hunk 1 above)
Comment (J-467) [reply pkg/azure/README.md]: Updated

**Review by smcclure20 — APPROVED** [review]

File: docs/source/developers/contributing/prerequisites.rst [hunk 2]
```diff
@@ -34,6 +34,10 @@ This is the list of core dependencies to install for the most common tasks. In g
     .. code-block:: console
     
         $ apt install build-essential
+
+      Then install ``make``:
+  
+        $ apt install make
```
Comment (seankimkdy) [inline docs/source/developers/contributing/prerequisites.rst:40]:
```suggestion
    Then install ``make``:
  
    .. code-block:: console

        $ apt install make
```

Spacing is a little tricky for RST. Right now, what you currently have looks like this

Screenshot 2024-07-04 at 00 01 07

I assume you want it to look like this.

Screenshot 2024-07-04 at 00 01 15

**Review by seankimkdy — CHANGES_REQUESTED** [review]

**Review by seankimkdy — APPROVED** [review]

**Pull request merged**